src/app.js

```diff
@@ -1 +1,34 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from "axios";
+
+async function fetchAllCountries() {
+    const BASE = "https://restcountries.com/"
+    const ENDPOINT = "v2/all"
+    try {
+        const response = await axios.get(BASE + ENDPOINT);
+        console.log(response);
+        const countries = response.data;
+
+        countries.sort((a, b) => {
+            return a.population - b.population
+        })
+        console.log( countries);
+
+        const listOfCountries = document.getElementById("listOfCountries")
```

Comment:
Voor de naamgeving van een id gebruiken we alleen lowercase en een streepje tussen de woorden niet camelcase
dus zoiets: list-of-countries 
Ik heb bij de HTML hetzelde gezet